index.html

```diff
@@ -139,18 +139,22 @@ <h1 class="heading-about-me">About me</h1>
         <form class="input" action="https://formspree.io/f/mnqrljzr"
         method="POST">
           <div class="name-container">
-          <label for="firstname">
-            <input type="text" class="item1" id="firstname" name="firstname" maxlength="30" placeholder="First Name" required>
+            <label for="firstname">
+            <small id="small-1"></small>
+            <input type="text" class="item1" id="firstname" name="firstname" maxlength="30" placeholder="First Name">
           </label>
           <label for="lastname">
-            <input type="text" id="lastname" class="item2" name="lastname" maxlength="30" placeholder="Last Name" required>
+            <small id="small-2"></small>
+            <input type="text" id="lastname" class="item2" name="lastname" maxlength="30" placeholder="Last Name">
           </label>
           </div>
           <label for="email">
-            <input type="email" id="email" class="item3" name="email" placeholder="Email address" required>
+            <small id="small-3"></small>
+            <input type="email" id="email" class="item3" name="email" placeholder="Email address">
           </label>
           <label for="text">
-            <textarea id="text" class="item4" name="textarea" maxlength="500" placeholder="Enter text here" rows="4" cols="50" required></textarea>
+            <small id="small-4"></small>
+            <textarea id="text" class="item4" name="textarea" maxlength="500" placeholder="Enter text here" rows="4" cols="50"></textarea>
```

Comment:
- [ ] Kindly add the `required` attribute to this textarea field to help with basic form validation